.github/workflows/salesloft.yml

```diff
@@ -0,0 +1,75 @@
+name: SalesLoft
+on:
+  push:
+    branches: [ master ]
+    tags: [ 'v*' ]
+  pull_request:
+
+jobs:
+  rspec:
+    name: RSpec
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:10.10
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
```

Comment:
because this gem doesn't have any dependencies that rely on the `rails` gem we shouldn't have to test any rails stuff in the github actions 🎉 So you can use a ruby matrix similar to the bussey gem: https://github.com/SalesLoft/bussey/blob/b56e32b67a114befe5bf278d8ded8ddebb063272/.github/workflows/salesloft.yml#L17